### ISSUE TITLE
html blocks from jupyter notebook md not rendered into database

### ISSUE THREAD

**Issue by Narkle** [Cobertos/md2notion#22]
When i use the standard import feature of notion for importing a jupyter notebook already converted into md, the html table block are rendered into a database block.

When i use md2notion, the html block are rendered into html code. 

It would be great to have the choice

Could you please append an option ?

**Comment by Cobertos** [comment]
After #21 got merged, we [do have the ability to convert HTML inside of the Markdown](https://github.com/Cobertos/md2notion/blob/master/md2notion/NotionPyRenderer.py#L381-L393), but currently only images are supported.

Can you post an example of the sort of table you're looking to convert?

If the project continues to go this way, it might be nice to have an HTML importer and then the Markdown HTML blocks could be passed to that as they are found, features could then just be added incrementally to the HTML importer.

**Comment by Cobertos** [comment]
Going to close this. Feel free to reopen a new issue but I think HTML conversion is outside the scope of this, even if Notion's MD importer does it, it should really be a separate thing.
